packages/manager/apps/pci-instances/src/pages/instances/create/components/Network.component.tsx

```diff
@@ -47,6 +64,50 @@ const Network: FC = () => {
     });
   };
 
+  const handleOpenCreateNetwork = () => {
+    setOpenCreateNetwork(true);
+
+    updatePrivateNetworkFields({
+      networkId: null,
+      newPrivateNetwork: ovhPrivateNetwork,
+    });
+
+    trackClick({
+      location: PageLocation.funnel,
+      buttonType: ButtonType.button,
+      actionType: 'action',
+      actions: ['add_instance', 'create_private_network'],
+    });
+  };
+
+  const handleCloseCreateNetwork = () => {
```

Comment:
Question: No tracking planned for closing the network creation ? Like 'create_private_network' 'cancel'